drop_server/backend/views.py

```diff
@@ -107,6 +110,10 @@ def get_if_modified_since(request):
     return (False, datetime.fromtimestamp(0)) if since is None else (True, dateparser.parse(since))
 
 
+def set_last_modified(response, modificationDate):
+    response.headers['Last-Modified'] = modificationDate.strftime("%a, %d %b %Y %H:%M:%S GMT")
```

Comment:
HTTP is a stupid protocol.

A drop message is saved with created_at in UTC. All times that are saved in PostgreSQL are in UTC and it is initialized with an explicit UTC-timestamp.

Since UTC = GMT in our case, I guess hard coding GMT is fine. In my opinion we should switch to ETAGs anyway.